runtime/doc/syntax.txt

```diff
@@ -2451,6 +2451,12 @@ have the following in your .vimrc: >
 
 	let filetype_m = "mma"
 
+MBSYNC					*mbsync.vim* *ft-mbsync-syntax*
+
+The mbsync application uses a configuration file called mbsyncrc to setup
+mailboxes names, user and password. All files ending with `mbsyncrc` will
```

Comment:
Do we need an additional dot?
```suggestion
mailboxes names, user and password. All files ending with `.mbsyncrc` will
```

Reply:
Rephrased the explanation. I included the `.` for mbsyncrc